reference/TubePlayer/TubePlayer/Business/YoutubeService.cs

```diff
@@ -3,10 +3,12 @@ namespace TubePlayer.Business;
 public class YoutubeService : IYoutubeService
```

Comment:
@ajpinedam can we make this a record and then simplify constructor+fields

Reply:
Wouldn't using a `record` force the two fields `_client` and `_playerClient` to be public?

Reply:
It's net8 now so c#12. It can be a class with default constructor.

Reply:
Updated using now C#12 Primary constructors.